### ISSUE TITLE
Add casting info on decimal query builder

### ISSUE THREAD

**Issue by gDelgado14** [knex/documentation#111]
# Environment

Knex version: ^0.13.0
Database + version: psql 9.6-2
OS: Docker Alpine

# Bug

Docks for Knex on schema building don't specify that psql `decimal` is casted to string. 

# Feature discussion / request

Add documentation in the `decimal` section to let users know that decimals are strings within the runtime.

**Comment by gDelgado14** [comment]
Maybe I wasn't clear on my description, but I was already aware that the created column type is actually a decimal. This is not news to me. 

However, you glossed over the main point that I was raising. Which is literally what you just said, "Decimal cannot be properly presented with javascript number". If you copy and paste that quote into the docs, you'll be providing useful information to users of psql who use knex. Otherwise they'll assume that decimal from psql is represented as a number in the js runtime.

**Comment by gDelgado14** [comment]
For reference:

https://stackoverflow.com/questions/45569216/knex-postgres-returns-strings-for-numeric-decimal-values

**Comment by kibertoad** [comment]
@gDelgado14 Yes, that part is clear, but as was already said, this has nothing to do with knex, this is a behaviour of specific postgresql driver. If you were using some custom postgresql driver that does implicit conversion to non-string, that documentation would have been wrong - again, for the reasons entirely unrelated to knex.

**Comment by elhigu** [comment]
Maybe just stating `Its driver issue.` wasn't clear enough. What I wanted to say is that we are not going to duplicate each drivers documentation about how they are treating their SQL column types in javascript side.
